civiclens/collect/check_doc_status.py

```diff
@@ -0,0 +1,69 @@
+"""This is to check the status of the documents in the database.
+- if the closing date has passed, the status should be updated
+- if the closing date is null or in the future:
+    - call the regulations.gov API to get the current status
+"""
+from datetime import datetime
+from civiclens.collect.move_data_from_api_to_database import (
+    connect_db_and_get_cursor,
+    query_register_API_and_merge_document_data,
+    insert_document_into_db)
+from civiclens.collect.access_api_data import pull_reg_gov_data
+from civiclens.utils.constants import REG_GOV_API_KEY
+
+def get_open_docs() -> list:
+    """
+    Get all documents that are open in the database.
+    Outputs:
+        open_docs (list): a list of tuples of the form (id, closing_date)
+    """
+    conn, cur = connect_db_and_get_cursor()
+    with conn:
+        cur.execute("SELECT id, comment_end_date \
+                    FROM regulations_document WHERE open_for_comment = %s", ('true',))
+        open_docs = cur.fetchall()
+    conn.close()
+    return open_docs
+
+def close_doc_comment(doc_id: str) -> None:
+    """
+    Update the status of a document in the database to closed.
+    Inputs:
+        doc_id (int): the id of the document to update
+
+    """
+    conn, cur = connect_db_and_get_cursor()
+    with conn:
+        cur.execute("UPDATE regulations_document \
+                    SET open_for_comment = %s \
+                    WHERE id = %s", ('false', doc_id))
+    conn.close()
+
+
+def check_current_status(open_docs: list) -> None:

```

Comment:
if the purpose of this function is the check the status, why do we insert the document into the database? Is the purpose just to ensure the document data is up to date? Do we know that document data actually changes over time

Reply:
If it's still open, we check on API to update the value. In case they closed/edited the documents before end date

Reply:
Does that actually happen?

Reply:
Yes in previous iteration, some comment didn't have an end date pulled from API. We have to check the status with calling the API

Reply:
ok sounds good